ecs-init/cache/cache_test.go

```diff
@@ -122,13 +123,18 @@ func TestDownloadAgentDownloadMD5Failure(t *testing.T) {
 
 	mockFS := NewMockfileSystem(mockCtrl)
 	mockGetter := NewMockhttpGetter(mockCtrl)
+	mockMetadata := NewMockinstanceMetadata(mockCtrl)
 
-	mockFS.EXPECT().MkdirAll(config.CacheDirectory(), os.ModeDir|0700)
-	mockGetter.EXPECT().Get(config.AgentRemoteTarballMD5()).Return(nil, errors.New("test error"))
+	gomock.InOrder(
+		mockFS.EXPECT().MkdirAll(config.CacheDirectory(), os.ModeDir|0700),
+		mockMetadata.EXPECT().Region().Return(config.DefaultRegionName, nil),
```

Comment:
I realize that the default path is the same as an unmapped region; I want that to be tested explicitly.  Please use a different return value than `config.DefaultRegionName`.